### PR TITLE
Copy json version of AerocomEvaluation configuration to experiment directory after processing

### PR DESCRIPTION
**Pull request by jgliss**
Tiny PR that will automatically copy config files into experiment directory after processing.

### CODE REVIEW

**Comment by jgliss** [comment]
@hansbrenna can you briefly review? You should run your upcoming CAMS61 analyses based on this update, so the online visualisation of the config files, that @AugustinMortier implemented now, is available.

**Comment by jgliss** [comment]
@hansbrenna could you please briefly review this PR?

**Comment by jgliss** [comment]
@ejgal would you quickly review this PR? Thank you.

File: pyaerocom/web/aerocom_evaluation.py [hunk 1]
```diff
@@ -1302,13 +1303,44 @@ def to_dict(self):
...
+        Returns
+        -------
+        str
+            name of config file
+        """
+        return '{}.json'.format(self.name_config_file)
+
     def to_json(self, output_dir):
-        """Convert configuration to json ini file"""
+        """Convert analysis configuration to json file and safe
```
Comment (ejgal) [inline pyaerocom/web/aerocom_evaluation.py:1331]:
```suggestion
        """Convert analysis configuration to json file and save
```

**Review by ejgal — APPROVED** [review]
Looks good! Great idea to store the config with the output.

**Comment by jgliss** [comment]
> Looks good! Great idea to store the config with the output.

Thanks!!

**Pull request merged**